src/bbb/tasks/bandit.py

```diff
@@ -117,51 +121,54 @@ def loss_step(self, X: torch.Tensor, y: torch.Tensor):
     hidden_layers = 2,
     hidden_units = 100,
     batch_size = 100,
-    lr = 1e-3,
+    lr = 1e-4,
     epochs = 100,
     early_stopping=False,
     early_stopping_thresh=1e-4
 )
 
 # Class for Greedy agents
 class Greedy(MushroomBandit):
-    def __init__(self, epsilon=0, lr=2e-5, **kwargs):
-        super().__init__(**kwargs)
+    def __init__(self, epsilon=0):
+        super().__init__()
         self.n_weight_sampling = 1
         self.epsilon = epsilon
-        self.net = RegressionDNN(params=DNN_REGRESSION_PARAMETERS).to(DEVICE)
-        self.optimizer = optim.SGD(self.net.parameters(), lr=lr)
+        self.net = DNN(params=DNN_REGRESSION_PARAMETERS).to(DEVICE)
         self.criterion = torch.nn.MSELoss()
         
     def loss_step(self, x, y, batch_id):
-        
         self.net.zero_grad()
         preds = self.net.forward(x)
         loss = self.criterion(preds, y)
         loss.backward()
-        self.optimizer.step()
+        self.net.optimizer.step()
         return loss
 
 BNN_REGRESSION_PARAMETERS = Parameters(
     name = "BBB_regression",
```

Comment:
This should be `BBB_rl`; also need to change `DNN_regression` to `DNN_rl`

Reply:
What do you mean by this one? The name of the parameters I guess?